src/appleseed/renderer/kernel/rendering/generic/genericframerenderer.cpp

```diff
@@ -350,6 +350,7 @@ namespace
                 if (!m_abort_switch.is_aborted())
                 {
                     m_frame.denoise(
+                        m_tile_renderers.size(),
```

Comment:
Although that's technically correct, wouldn't `m_params.m_thread_count` make more sense?